src/Dialog/Dialog.js

```diff
@@ -331,7 +335,7 @@ class DialogInline extends Component {
               className={contentClassName}
               style={styles.content}
             >
-              <Paper zDepth={4}>
+              <Paper zDepth={4} style={prepareStyles(styles.paper)}>
```

Comment:
No need for `prepareStyles`. We use that help before passing the style to a raw dom element.
It could be as simple as the following. Making that PR a two line changes.
```jsx
<Paper zDepth={4} style={paperStyle}>
```